src/bin/pgcopydb/cli_common.c

```diff
@@ -1010,6 +1034,20 @@ cli_copy_db_getopts(int argc, char **argv)
 				break;
 			}
 
+			case 'u':
+			{
+				if (!stringToInt(optarg, &options.splitMaxParts) ||
+					options.splitMaxParts < 1 ||
+					options.splitMaxParts > 128)
```

Comment:
Why is the maximum number of partitions 128? Sounds reasonable to me, but I think it warrants a comment in the code and also documentation coverage.

Reply:
Now that you mention it, it does not make sense to set an upper bound at all. Maybe I should just remove the upper bound.